src/Command/Field/InfoCommand.php

```diff
@@ -1,10 +1,5 @@
 <?php
 
-/**
```

Comment:
No longer requiring `@file` is based on DrupalCoding standards this app must be PSR2 compatible